### ISSUE TITLE
[R&D] explore adding webhints?

### ISSUE THREAD

**Issue by swyxio** [jaredpalmer/tsdx#198]
never seen this before but saw this in a convo between TS team members:

https://webhint.io/docs/user-guide/hints/hint-typescript-config/

**Comment by jaredpalmer** [comment]
This seems very useful

**Comment by jaredpalmer** [comment]
However, maybe we should start with just documenting it. Perhaps in a "suggested packages" section of the docs?

**Comment by sw-yx** [comment]
yeah cool. we've yet to discuss this at all but what are your thoughts on adding this to the github readme vs the doc site content? duplication warranted? or consolidate on one or the other? your call im just happy to help. i agree this thing is pretty useful esp since it seems officially endorsed.
